.github/workflows/ci.yml

```diff
@@ -114,6 +114,5 @@ jobs:
           MODULES: ${{ needs.setup-module-test.outputs['test-modules'] }}
           AWS_ACCESS_KEY_ID: ${{ secrets[env.MATRIX_AWS_ACCESS_KEY_ID] }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets[env.MATRIX_AWS_SECRET_ACCESS_KEY] }}
-          IRONPLANS_TOKEN: ${{ secrets.IRONPLANS_TOKEN }}
```

Comment:
that was an old token for auth0 authentication which is only needed if authentication is on. this test has authentication off. there was actually a bug in `simple-integration` (`AO` instead of `A0`) which that made it so that it never tested the authentication path so they IP tokens was completely ignored / misused